Library/Homebrew/formula_installer.rb

```diff
@@ -311,12 +334,13 @@ def inherited_options_for(dep)
   end
 
   def install_dependencies(deps)
-    if deps.length > 1
+    if deps.empty? && only_deps?
+      puts "All dependencies for #{formula.full_name} are satisfied."
```

Comment:
That print is from the original `compute_and_install_dependencies`, so it wasn't introduced by me. I can take it out if you'd like, though.